### ISSUE TITLE
LOBS.ART - a terminal for artist drop on LOBs

### ISSUE THREAD

**Issue by ekius** [lobster-dao/overview#18]
**TLDR:** I would grant 1 LOB to make smartcontract/ site, which help an artist do a permissionless drop on LOB holders.
---
Details below:
 
**Context:**
With the LOBS revealing is coming close. As much as many people want to contribute for the LObs DAO, I want to make an Generative art drop on LOBs Holders. But I don't know where to ask, or which dev should i trust. (I'm such a pleb)

**Proposal:**
I would grand 1 LOB for any person or team reaching out and do : 

- A smartcontract template that ultilise all the data of LOBS traits for minting. 
- A UI/UX for artist to do bulk minting (both JPG and metadata) which integrates with traits data above.
- A UI/UX for people to claim or at least searching for what they could mint with LOBs in their wallet.

Some specs i think would be needed for a future drops:
- Royalty (0-10% for artist, 2.5% for LOBs treasury, 0.1% for SC audited board/person )
- Mint fee (may be less than 0.1 ETH )
- Bulk minting (as many holders want to save gas)
- An audited tick from a trusted person, who will get the 0.1% above to help artists. 

**Some other ideas (optional) :**  
- Auction mint.
- Secret mint for specific LOBs.
- Decaying mint (say like if the drop isn't minted it in specific time, then it can't be minted anymore )

This is not a community proposal as i would grant my LOBs for anyone can help. Please reaching out to me in NFT avenue if needed.
